models/Program.js

```diff
@@ -168,7 +169,7 @@ Program.schema.virtual('isLiveNow').get(function () {
 	// Check biweekly first, then do a general bounds check.
 	if (this.isBiweekly) {
 		const curWeek = Math.floor(Math.floor(daysSinceBeginningOfYear() / 7) / 2);
-		const state = (curWeek == 0) ? true : false;
```

Comment:
Oh that's ok, I don't really care about stuff like that. I just happened to see it.

I haven't used isLiveNow yet so either way is fine.